ArduCopter/mode_follow.cpp

```diff
@@ -26,12 +26,15 @@ bool Copter::ModeFollow::init(const bool ignore_checks)
 
 void Copter::ModeFollow::run()
 {
-    // if not auto armed or motor interlock not enabled set throttle to zero and exit immediately
-    if (!motors->armed() || !ap.auto_armed || !motors->get_interlock()) {
-        zero_throttle_and_relax_ac();
+    // if not armed set throttle to zero and exit immediately
+    if (!motors->armed() || !ap.auto_armed || ap.land_complete) {
+        make_safe_spool_down();
```

Comment:
wp_nav->wp_and_spline_init(); ?

Reply:
It wouldn't be the wp_and_spline_init() but it could be that we should be calling pos_control->init_vel_controller_xyz();


I think you're probably correct that it is a problem but I'm probably not going to address it as part of this PR because in eithe case it would be an existing problem because we don't call the init in master either.  The way this problem would show up is as a very sudden start of the vehicle.  I.e. it wouldn't be acceleration limited I think.